src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +75,10 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge
+                        && p.getAge() <= (p.getSex() == Person.Sex.MAN ? maleToAge : femaleToAge))
```

Comment:
*Optionally: p.getAge() >= fromAge could be used once in spite of using it in both statements

Reply:
What do you mean once? I need to check max and min age

Reply:
You can use one more filter
```suggestion
                .filter(p -> p.getAge() >= fromAge)
                 .filter(p -> p.getAge() <= (p.getSex() == Person.Sex.MAN ? maleToAge : femaleToAge))
```